### PR TITLE
Add Construct query case to current ADT model

### PR DESCRIPTION
**Pull request by shihch**

### CODE REVIEW

File: src/main/scala/QueryConstruct.scala [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+      toStringVal(triple.getSubject),
+      toStringVal(triple.getPredicate),
+      toStringVal(triple.getObject)
+    )
+  }
+
+
+
+  def main(args: Array[String]): Unit = {
+//    val q = Source.fromFile("src/test/resources/queries/q3-union.sparql").mkString
```
Comment (puremachinery) [inline src/main/scala/QueryConstruct.scala:75]: Any reason not to remove commented code?

**Review by puremachinery — CHANGES_REQUESTED** [review]
Remove commented line of code, otherwise seems good.

Other note: John recently checked in scalastyle - might be good to run that over this, but not required for approval.

File: src/main/scala/StringLike.scala [hunk 2]
```diff
@@ -14,5 +14,6 @@ object StringVal {
   final case class STRING(s:String) extends StringVal
   final case class VARIABLE(v:String) extends StringVal
   final case class URIVAL(u:String) extends StringVal
+  final case class BLANK(ref:String) extends StringVal
```
Comment (JNKHunter) [inline src/main/scala/StringLike.scala:17]: Sweet! Was going to add Blank node support :)

**Review by JNKHunter — CHANGES_REQUESTED** [review]
Looks good Clark. There were some Scalastyle warnings (You can look at the 'checks' tab). If we could clear those out, I'm good to merge. We'll need to figure out how to have checks fail when scalastyle has warnings.

File: src/main/scala/QueryConstruct.scala (hunk 1 above)
Comment (shihch) [reply src/main/scala/QueryConstruct.scala]: The main method is removed, which is no need for now.

**Comment by shihch** [comment]
scala style check is now passed.

**Review by puremachinery — APPROVED** [review]
Looks like this covers the comment and the scalastyle changes. Great.

**Review by JNKHunter — APPROVED** [review]

**Pull request merged**